lr-cli/cortx_setup/commands/node/finalize.py

```diff
@@ -38,7 +38,7 @@ def _validate_health_map(self):
         self.logger.debug("Validating node health check")
         resource_paths = [HEALTH_PATH, MANIFEST_PATH]
         for resource in resource_paths:
-            path = get_pillar_data(resource)
+            path = get_pillar_data(resource).split('://')[1]
```

Comment:
can  you add this split after below check, if path is present then only do it.

Reply:
Yes. i have moved the split after the path check..